Structure_Engine/Query/MaterialType.cs

```diff
@@ -36,41 +34,59 @@ public static partial class Query
         /**** Public Methods                            ****/
         /***************************************************/
 
+        [Description("Gets the material type from the material fragment. This will for a Steel material always return type Steel.")]
```

Comment:
Material
I think the second sentence should be e.g.

Reply:
Not sure we should capitalise the material here. This refers to the structures oM where we do not have any class named Material